cadc-download-manager-server/src/main/resources/META-INF/resources/chooser.jsp

```diff
@@ -3,7 +3,7 @@
 *******************  CANADIAN ASTRONOMY DATA CENTRE  *******************
 **************  CENTRE CANADIEN DE DONNÉES ASTRONOMIQUES  **************
 *
-*  (c) 2020.                            (c) 2020.
+*  (c) 2009, 2020.                      (c) 2009, 2020.
```

Comment:
Strange.  The year was reverted here.

Reply:
I liked the idea of keeping a history of files like this. It's older code with newer updates. This syntax might not be correct, though?

Reply:
GitHub made it look like that 2020 was _replaced_ by 2009.  We don't do this anywhere else that I've seen.  That explains the commas I've been seeing.  Did you do this on every file you touched?